blocks/api/raw-handling/inline-block-corrector.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Internal dependencies
+ */
+import { isInlineBlock } from './utils';
+
+/**
+ * Browser dependencies
+ */
+const { ELEMENT_NODE } = window.Node;
+
+export default function( node ) {
+	if ( node.nodeType !== ELEMENT_NODE ) {
+		return;
+	}
+
+	if ( ! isInlineBlock( node ) ) {
+		return;
+	}
+
+	let wrapper = node;
+
+	while ( wrapper.nodeName !== 'P' ) {
```

Comment:
I think you might need `while( wrapper && wrapper.nodeName )` here, because the final iteration will be the root HTML node with `parentElement` = `null`.

Reply:
Thanks! I changed it from `while ( wrapper.nodeName !== 'P' && ( wrapper = wrapper.parentElement ) ) {}` and forgot to add the check. 🙈 Nice catch.